packages/www/package.json

```diff
@@ -23,5 +23,12 @@
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "theme-ui": "^0.3.1"
+  },
+  "devDependencies": {
```

Comment:
Should these deps live at the root level `package.json`, or be repeated in `www-partycorgi-me/package.json`? Still learning how monorepos work.